scipy/special/tests/test_boost_ufuncs.py

```diff
@@ -42,3 +43,19 @@ def test_stats_boost_ufunc(func, args, expected):
             value = func(*args)
         assert isinstance(value, typ)
         assert_allclose(value, expected, rtol=rtol)
+
+
+def test_landau():
```

Comment:
Where should this go? The intent is not to check accuracy - just that the function exists and is wrapped as expected, really.

Reply:
The location and tests look fine to me. Let's rely on Boost for accuracy testing. Just add that as a comment?